adserver/tasks.py

```diff
@@ -478,12 +481,15 @@ def notify_of_publisher_changes():
         previous_week_report.generate()
 
         for metric in ("views", "ctr"):
+            total_views = (
+                last_week_report.total["views"] + previous_week_report.total["views"]
+            )
             last_week_value = last_week_report.total[metric]
             previous_week_value = previous_week_report.total[metric]
             if last_week_value > 0 and previous_week_value > 0:
                 metric_diff = abs((last_week_value / previous_week_value) - 1)
-                perc_diff = (last_week_value / previous_week_value) * 100
-                if metric_diff > difference_threshold:
+                perc_diff = ((last_week_value / previous_week_value) - 1) * 100
```

Comment:
I imagine there must be a utility function to calculate this in the stdlib. Would be useful to add one if not, since we do this a few places and likely mess it up in other places :)

Reply:
I checked `math` and `statistics` but I don't see anything.

Reply:
I can put it into our own utilities.